src/services/auth.service.js

```diff
@@ -0,0 +1,143 @@
+import { Account } from "../models/account.model.js";
+import { AccountRepo } from "../repositories/account.repo.js";
+import { SessionRepo } from "../repositories/session.repo.js";
+import { hashPassword } from "../utils/hashPassword.js";
+import { generateAccessToken, generateRefreshToken } from "../jwt/jwt.js";
+import { verifyPassword } from "../utils/verifyPassword.js";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+
+export class AuthService {
+  //register
+  static async register(account) {
+    const hashedPassword = await hashPassword(account.password);
+
+    const entity = new Account(
+      null,
+      account.name,
+      account.email,
+      hashedPassword
+    );
+
+    const savedAccount = await AccountRepo.createAccount(entity);
```

Comment:
don't use prefix saved

```suggestion
    const account = await AccountRepo.createAccount(entity);
```